object-manipulation-1/src/set-value.js

```diff
@@ -0,0 +1,5 @@
+/* exported setValue */
+function setValue(object, key, value) {
+  object[key] = value;
+  return setValue;
```

Comment:
You don't want to `return setValue`, because `setValue` is the function we are inside. You don't want to `return` a function. You actually shouldn't even be `return`ing anything. It says that the function should not `return` anything.